docs/how-to-guides/install-seven-add-on.md

```diff
@@ -0,0 +1,189 @@
+---
+myst:
+  html_meta:
+    "description": "How to install a Seven frontend add-on with Cookieplone"
+    "property=og:description": "How to install a Seven frontend add-on with Cookieplone"
+    "property=og:title": "Install Seven with Cookieplone"
+    "keywords": "Plone, Plone 6, Volto, create, project, install, Cookieplone"
+---
+
+
+(install-cookieplone-label)=
+
+# Install Seven with Cookieplone
+
+This chapter describes how you can create a web application using the {term}`Cookieplone` template.
+
+This template is the recommended way to start a new add-on using the Seven frontend.
+It also includes tools for development and deployment.
+
+```{seealso}
+For other installation options, see {ref}`get-started-install-label`.
+```
+
+(create-project-cookieplone-system-requirements)=
+
+## System requirements
+
+Plone and Seven have both hardware requirements and software prerequisites.
+
+
+### Supported web browsers
+
+```{include} /volto/_inc/_install-browser-reqs-volto.md
+```
+
+```{include} /_inc/_install-browser-reqs-classic-ui.md
+```
+
+
+(create-project-cookieplone-hardware-requirements-label)=
+
+### Hardware requirements
+
+```{include} /_inc/_hardware-requirements.md
+```
+
+
+(create-project-cookieplone-prerequisites-for-installation-label)=
+
+### Prerequisites for installation
+
+```{include} ../volto/_inc/_install-operating-system.md
+```
+
+-   {term}`uv`
+-   {term}`nvm`
+-   {term}`Node.js`
+-   {term}`GNU make`
+-   {term}`Git`
+
+
+#### uv
+
+```{include} /_inc/_install-uv.md
+```
+
+
+#### Python
+
+{term}`uv` will download and install the required Python version for your project.
+
+
+#### nvm
+
+```{include} ../volto/_inc/_install-nvm.md
+```
+
+
+#### Node.js
+
+```{include} ../volto/_inc/_install-nodejs.md
+```
+
+3.  Enable {term}`corepack` so that Node.js will install {term}`pnpm` as a package manager.
+
+    ```shell
+    npm i -g corepack@latest && corepack enable
+    ```
+
+
+#### Make
+
+```{include} ../volto/_inc/_install-make.md
+```
+
+
+#### Git
+
+```{include} ../volto/_inc/_install-git.md
+```
+
+
+(create-project-cookieplone-generate-the-project-label)=
+
+## Generate the add-on project
+
+After satisfying the prerequisites and having activated an LTS version of Node,
+generate the add-on project.
```

Comment:
```suggestion
After satisfying the prerequisites and having {ref}`activated an LTS version of Node.js <seven-prerequisites-for-installation-nodejs-label>`, generate the add-on project.
```

Reply:
Done in 5e437a39bfae9297babdf89e4399b8cf46f6ce83. See also https://github.com/plone/documentation/pull/1929.